server/message_hooks.go

```diff
@@ -38,27 +38,32 @@ func (p *Plugin) UserWillLogIn(_ *plugin.Context, user *model.User) string {
 }
 
 func (p *Plugin) MessageHasBeenPosted(_ *plugin.Context, post *model.Post) {
+	p.GetMetrics().ObserveMessageHooksEvent("message_create")
```

Comment:
Counting here will always be out of sync with the shared channels system, since we get the event for all messages whether or not we process them, whereas shared channels is only for things to which we subscribe. Should we try to push this metric down further?

Reply:
Yes, I'll take a look

Reply:
Done